todocore/src/test/java/com/knighttodo/todocore/integration/DayTodoResourceIntegrationTest.java

```diff
@@ -328,5 +328,4 @@ public void findDayTodosByDayId_shouldReturnExistingTodo_whenIdIsCorrect() throw
                 .andExpect(status().isFound())
                 .andExpect(jsonPath(buildJsonPathToLength()).value(2));
     }
-
-}
+}
```

Comment:
You shouldn't change something like this, it's additional file in the PR, it's redundant for the task.

Reply:
Yes. It's redundant for this task. But this test works only with these changes.